contracts/seed/Seed.sol

```diff
@@ -316,32 +283,46 @@ contract Seed {
     /**
       * @dev                     Close distribution.
     */
-    function close() public onlyAdmin isActive {
+    function close() external onlyAdmin {
```

Comment:
Can we reword the function description comment to read "Shut down contributions (buying).  Supersedes the normal logic that eventually shuts down buying anyway.  Also shuts down the admin's ability to alter the whitelist."

Reply:
Should I this whole sentence?

Reply:
yes please

Reply:
done